### ISSUE TITLE
Support for Horovod

### ISSUE THREAD

**Issue by xiaoyongzhu** [wbuchwalter/tensorflow-k8s-azure#38]
https://github.com/uber/horovod is an open source framework by Uber for easier TensorFlow/Keras job distributed training, and is adopted by Azure Batch AI. Looks like it's easy and it would be awesome if we can integrate that part in this k8s tutorial. 

One of the biggest advantage is that it requites minimal code changes.
